.travis.yml

```diff
@@ -35,13 +35,4 @@ install:
 script:
   - tox
 
-matrix:
-  allow_failures:
-    - env: TOXENV=py27-django110
-    - env: TOXENV=py34-django110
-    - env: TOXENV=py35-django110
-    - env: TOXENV=py27-django_trunk
```

Comment:
I'd rather leave the trunk builds in `allow_failures` even if they pass currently. Once we get into the heat of 1.11 feature development, they will probably break again, and I don't want the build breaking due to changes outside model-utils' control. Django trunk build is just advance warning, it's not a blocker.